src/data/finsupp/basic.lean

```diff
@@ -770,6 +770,14 @@ lemma _root_.submonoid.finsupp_prod_mem (S : submonoid N) (f : α →₀ M) (g :
   (h : ∀ c, f c ≠ 0 → g c (f c) ∈ S) : f.prod g ∈ S :=
 S.prod_mem $ λ i hi, h _ (finsupp.mem_support_iff.mp hi)
 
+lemma finsupp.prod_mul_distrib [has_zero M] [comm_monoid N] {f : α →₀ M} (g1 g2 : α → M → N) :
+  f.prod (g1 * g2) = (f.prod g1) * (f.prod g2) :=
+by simp [finsupp.prod, finset.prod_mul_distrib]
```

Comment:
This exists already, it's `finsupp.prod_mul`

Reply:
Sorry, I can't find `finsupp.prod_mul`.  I can see `dfinsupp.prod_mul` which does something very similar in `data/dfinsupp`, but that file isn't imported by `data/finsupp/basic`.  Am I misunderstanding something?

Reply:
Sorry, don't know how I missed this.